server/sonar-server/src/test/java/org/sonar/server/batch/ProjectRepositoryLoaderMediumTest.java

```diff
@@ -453,7 +453,7 @@ public void return_quality_profile_from_project_profile() throws Exception {
     QualityProfileDto profileDto = QProfileTesting.newDto(QProfileName.createFor(ServerTester.Xoo.KEY, "SonarQube way"), "abcd").setRulesUpdatedAt(
       DateUtils.formatDateTime(ruleUpdatedAt));
     tester.get(DbClient.class).qualityProfileDao().insert(dbSession, profileDto);
-    tester.get(DbClient.class).propertiesDao().setProperty(new PropertyDto().setKey("sonar.profile.xoo").setValue("SonarQube way").setResourceId(project.getId()), dbSession);
+    tester.get(DbClient.class).qualityProfileDao().insertProjectProfileAssociation(project.uuid(), profileDto.getKee(), dbSession);
 
```

Comment:
Instead of updating this big Medium tests, removing these test cases and create a new ProjectAssociationActionsTest using DbTester and EsTester